mocks/log.go

```diff
@@ -126,47 +126,47 @@ func (msw *MockSyslogWriter) Clear() {
 }
 
 // Close releases resources. No other methods may be called after this.
-func (msw *MockSyslogWriter) Close() error {
```

Comment:
So, with the pointer receiver, I got "MockSyslogWriter doesn't implement SyslogWriter (method XXX has pointer receiver)" errors. So this was a fix for that. If there is a better way to fix that issue, please let me know!